### ISSUE TITLE
in dials.export: dqe scaling should be applied to variance not standard deviation / sigI

### ISSUE THREAD

**Issue by graeme-winter** [dials/dials#239]
Random probability (uniform deviate) of measuring a photon leaves a poisson distribution so the variance should be scaled not the sigI in the export process.

**Comment by graeme-winter** [comment]
Have added the changes and run tests, all still pass

Taking some small molecule data through to structure solution the effect of this seems to be very modest...

**Comment by Anthchirp** [comment]
Reverted following this discussion

@graeme-winter 
I looked into this some - see cctbx/modules/dials_scratch/gw/poisson_dqe.py - will welcome input from learned colleagues whether this is correct or no
Not sure if this change is really right... experiment suggests it is but not obvious - DGW RJG thoughts?
Reason behind this BTW is the effect is in the measurement process not in the source, so the outcome of *this* scaling should remain poissonian (sp?) i.e. var(I) ~ I - retained by scaling I and varI by 1/LP
*wrong*
1/DQE
I, sigI still scaled by LP
... discuss

@dagewa 
The distribution of a scaled Poisson distribution is no longer itself Poisson
that this is so is obvious from the fact that the *shape* of the Poisson distribution changes with its rate constant

GW:
What is right then?

DW:
Imagine a detector with extremely poor DQE. Let's say it records only 1% of photons. You want to scale that data by 100 to get an approximation of the real mean
you must scale the variance by 100^2

GW:
Ok like it was

DW:
then the poor error of the 1% DQE detector is still reflected in the rescaled quantity
otherwise there would be no advantage in buying a 100% DQE detector!

GW:
Commit above probably wants reverting then...
